test/unit/test_paired_loader.py

```diff
@@ -0,0 +1,136 @@
+"""
+Tests functionality of the PairedDataLoader
+"""
+import pytest
+
+from deepreg.dataset.loader.h5_loader import H5FileLoader
+from deepreg.dataset.loader.paired_loader import PairedDataLoader
+
+# assign values to input vars
+data_dir_path = "./data/test/h5/paired/test"
+sample_label = "sample label"
+seed = 42
+moving_image_shape_arr = (999, 999, 999)
```

Comment:
I'm wondering maybe smaller values will accelerate the tests?
I suggest (16, 16, 16) or (8, 8, 8)